solidity/contracts/Zoltar.sol

```diff
@@ -117,12 +116,12 @@ contract Zoltar {
 		return marketResolutionData.outcome;
 	}
 
-	function migrateStakedRep(uint192 _universeId, uint56 _marketId, uint8 _outcome) external {
+	function migrateStakedRep(uint192 _universeId, uint56 _marketId) external {
 		MarketResolutionData memory marketResolutionData = marketResolutions[_universeId][_marketId];
 		require(marketResolutionData.reportTime != 0, "No REP staked in this market");
 		require(!marketResolutionDataIsFinalized(marketResolutionData), "Cannot migrate REP from finalized market");
 
-		migrateREPInternal(_universeId, REP_BOND, _outcome, address(this), marketResolutionData.initialReporter);
+		migrateREPInternal(_universeId, REP_BOND, address(this), marketResolutionData.initialReporter, 3);
```

Comment:
Consider using a more obvious sentinel value for this (and make the associated change in the loop below).  This will help make it more clear to the reader that this is a sentinel value.
```suggestion
		migrateREPInternal(_universeId, REP_BOND, address(this), marketResolutionData.initialReporter, type(uint8).max);
```